app/models/prediction.rb

```diff
@@ -22,48 +22,44 @@ def self.parse_deadline(date)
 
   DEFAULT_INCLUDES = [:judgements, :responses, :creator].freeze
 
-  def self.unjudged(limit: 100)
+  def self.unjudged
     not_private
       .not_withdrawn
       .includes(DEFAULT_INCLUDES)
       .where('(SELECT outcome AS most_recent_outcome FROM judgements WHERE prediction_id = predictions.id ORDER BY created_at DESC LIMIT 1) IS NULL AND deadline < UTC_TIMESTAMP()')
       .order(deadline: :desc)
-      .limit(limit)
   end
 
-  def self.judged(limit: 100)
+  def self.judged
     not_private
       .not_withdrawn
       .includes(DEFAULT_INCLUDES)
       .joins(:judgements)
       .where('(SELECT outcome AS most_recent_outcome FROM judgements WHERE prediction_id = predictions.id ORDER BY created_at DESC LIMIT 1) IS NOT NULL')
       .order('judgements.created_at DESC')
-      .limit(limit)
   end
 
-  def self.future(limit: 100)
+  def self.future
     not_private
       .not_withdrawn
       .includes(DEFAULT_INCLUDES)
       .where('(id NOT IN (SELECT prediction_id FROM judgements) OR id IN (SELECT prediction_id FROM judgements WHERE outcome IS NULL)) AND deadline > UTC_TIMESTAMP()')
       .order(:deadline)
-      .limit(limit)
   end
 
-  def self.recent(limit: 100)
-    order(created_at: :desc).not_private.not_withdrawn.includes(DEFAULT_INCLUDES).limit(limit)
+  def self.recent
+    order(created_at: :desc).not_private.not_withdrawn.includes(DEFAULT_INCLUDES)
   end
 
-  def self.popular(limit: 100)
+  def self.popular
     not_private
       .not_withdrawn
       .includes(:responses, :creator)
       .joins(:responses)
       .where('predictions.deadline > UTC_TIMESTAMP() AND predictions.created_at > ?', 2.weeks.ago)
       .order('count(responses.prediction_id) DESC, predictions.deadline ASC')
       .group('predictions.id')
-      .limit(limit)
-      .select(&:unknown?)
+      # .select(&:unknown?) # Commented out because this is causing an error when I take out .limit(limit)
```

Comment:
So it turns out that the reason this is happening is because the call to `.select` is turning the result into an array rather than retaining it as an `ActiveRecord::Relation`. This means that the array doesn't know what to do with further scopes tacked on afterwards (like `limit` or `page`).